### PR TITLE
feat(core): Role display name

### PR DESCRIPTION
**Pull request by bodnara**
* Added property display name for roles
* Set up display names for roles in configuration
* Added display name to RoleManagementRules

### CODE REVIEW

File: perun-openapi/openapi.yml [hunk 1]
```diff
@@ -730,6 +731,7 @@ components:
         - entitiesToManage
         - assignedObjects
         - assignableToAttributes
+        - displayName
```
Comment (Johaney-s) [inline perun-openapi/openapi.yml:734]: will this work if display name is null?

Comment (xflord) [reply perun-openapi/openapi.yml:734]: works for primaryObject so I suppose it should work for displayName as well?

File: perun-base/src/main/resources/perun-roles.yml [hunk 2]
```diff
@@ -7822,6 +7822,7 @@ perun_policies:
 #  assignable_to_attributes is a flag that determines whether the role can appear in attribute policies.
 #  mfa_critical_role is a flag marking roles always requiring MFA from users having that role
 #  skip_mfa is a flag that whether the role should skip MFA check.
+#  display_name is a more user-friendly name
```
[the remark this answers is not in the document]
Comment (xflord) [reply perun-base/src/main/resources/perun-roles.yml:7825]: ++

File: perun-openapi/openapi.yml (hunk 1 above)
Comment (Johaney-s) [reply perun-openapi/openapi.yml:734]: right, missed that

File: perun-base/src/main/resources/perun-roles.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (bodnara) [reply perun-base/src/main/resources/perun-roles.yml:7825]: Javadoc was added to both `RoleManagementRules.java` and Ansible templates.

**Review by xflord — APPROVED** [review]

**Pull request merged**